__tests__/intergration.test.js

```diff
@@ -87,3 +87,40 @@ describe('GET /api/aricles', () => {
     })
   });
 });
+describe('GET /api/articles/:article_id/comments', () => {
+  test('200: returns all comments for provided article parameter, by most reacent', () => {
+    return request(app)
+    .get('/api/articles/9/comments')
+    .expect(200)
+    .then(({body}) => {
+        expect(body.comments.length).toBe(2)
+        expect(body.comments).toBeSortedBy('created_at', {descending: true})
+        body.comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id : expect.any(Number),
+            votes : expect.any(Number),
+            created_at : expect.any(String),
+            author : expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number)
+          })
+        })})
+    })
+    test('200: empty array when article has no comments', () => {
+      return request(app)
+      .get('/api/articles/2/comments')
+      .expect(200)
+      .then(({body}) => {
+          expect(body.comments.length).toBe(0)
+          expect(body.comments).toEqual([])
+      })
+  })
+  test('400: returns error when article_id does not exist', () => {
```

Comment:
This should be a different error code that is tested for as the article is not found